shared/app/routes-login.js

```diff
@@ -10,6 +10,7 @@ import loginRoutes from '../login/routes'
 const loginRouteTree = makeRouteDefNode({
   tags: makeLeafTags({title: loginRouteTreeTitle}),
   children: {
+    // $FlowIssue
```

Comment:
What's the problem?

Reply:
i added more comments but routetree typing is pretty broken